core/domain/email_manager.py

```diff
@@ -934,3 +938,92 @@ def send_test_email_for_bulk_emails(tester_id, email_subject, email_body):
     _send_email(
         tester_id, tester_id, feconf.BULK_EMAIL_INTENT_TEST,
         email_subject, email_body, tester_email, sender_name=tester_name)
+
+
+def send_mail_to_onboard_new_reviewers(user_id, category):
+    """Sends an email to users who have crossed the threshold score.
+
+    Args:
+        user_id: str. The ID of the user who is being offered to become a
+            reviewer.
+        category: str. The category that the user is being offered to review.
+    """
+
+    email_subject = 'Invitation to review suggestions'
+
+    email_body_template = (
+        'Hi %s,<br>'
+        'You have been actively contributing quality suggestions for Oppia '
+        'content in category %s. In recognition of your contributions, we would'
+        ' like to have you as one of our reviewers. As a reviewer, you would be'
+        ' responsible for regularly reviewing the suggestions in the same '
+        'category and ensuring that the accepted content meets the quality bar.'
+        ' If you would like to help out the '
+        'content creators to review suggestions, please visit your '
+        '<a href="https://www.oppia.org/creator_dashboard/">dashboard</a>. '
+        'And set the review preference for the category as yes. '
+        '<br>Note that if you do accept, you will recieve periodic '
+        'mails to review incoming suggestions.<br>'
+        'Looking forward to working with you! '
+        'Thanks!<br>'
+        '- The Oppia Team<br>'
+        '<br>%s')
+
+    if not feconf.CAN_SEND_EMAILS:
+        log_new_error('This app cannot send emails to users.')
+        return
+
+    recipient_user_settings = user_services.get_user_settings(user_id)
+    can_user_receive_email = user_services.get_email_preferences(
+        user_id).can_receive_email_updates
+
+    if can_user_receive_email:
+        # Send email only if recipient wants to receive.
+        email_body = email_body_template % (
+            recipient_user_settings.username, category, EMAIL_FOOTER.value)
+        _send_email(
+            user_id, feconf.SYSTEM_COMMITTER_ID,
+            feconf.EMAIL_INTENT_ONBOARD_REVIEWER,
+            email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_mail_to_notify_users_to_review(user_id, category):
+    """Sends an email to users to review suggestions in categories they have
+    agreed to review for.
+
+    Args:
+        user_id: str. The id of the user who is being pinged to review
+            suggestions.
+        category: str. The category of the suggestions to review.
+    """
+
+    email_subject = 'Notification to review suggestions'
+
+    email_body_template = (
```

Comment:
Ditto, this needs copywriting. Please include in a doc.

Reply:
@seanlip is the doc fine? If everything else also looks OK to you, could you please approve the PR?

Reply:
The doc is fine, but the changes I made in the doc do not seem to have been propagated to this PR.

Reply:
Hi @seanlip, in the doc you had mentioned to add links to the creator dashboard in the email, which is already there in the template here (I had removed links in the doc to focus on the content). Also, these emails aren't being sent yet (this PR just adds templates). The follow up PR will add code to send these emails (#5387, needs to be updated after this PR is merged).
So the comment about whether we should restrict who to send these emails to needs to be resolved, but I think that can happen in a later PR.

Reply:
I updated the text of the emails in the doc. These textual changes don't seem to be in this PR?

I agree that the thing about restricting who to send emails to can be done in a later PR.

Reply:
Oh I'm really sorry, I hadn't noticed that the email text had changed! I was focussing on the comments you had made. Will update this PR right now. Sorry for the confusion.

Thanks